static/docs/command-reference/checkout.md

```diff
@@ -95,6 +95,10 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - recreates links or copies for all checked out files even ones
```

Comment:
move it after `-R` in this section. At least after common the `h`, `q`, `v`

Reply:
I don't understand the logic. We have checkout specific options now, generic later. You want to mix it? I.e. specific, generic, specific again.

Reply:
my bad, sorry :) but it makes sense to move it after `-R` to keep them in the same order as we have them in CLI.

Reply:
What do you mean by CLI? If it's the usage section at the top then it has completely different order anyway, we'll need to reorder all of them either here or in usage section.

If you mean `dvc checkout --help` then it also has completely different order )

Reply:
usage section is in the same exact order as CLI, right? CLI itself is consistent.

In docs we don't put `h, q v` section on the top since they do not provide value (at some point we didn't have them at all described). Besides those, order should be the same as in CLI. And it's a bug that `-f` is out of order now. Most likely we have a ticket to fix them all. So, would really appreciate if we reoder `--relink` (as well as `-f` if it's not too hard :) ).

Reply:
@Suor are you going to address this or I can merge and fix it after?

Reply:
I was going to fix it in all three places, but got caught up with other stuff. So let's make it wait until tomorrow.

Reply:
👍 sure, np! thanks.